src/Server_Rendering_TEST.cc

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gtest/gtest.h>
+#include <gz/rendering/RenderEngine.hh>
+#include <gz/rendering/RenderingIface.hh>
+#include <gz/sim/rendering/Events.hh>
+#include <gz/rendering/Scene.hh>
+
+#include "gz/sim/Server.hh"
+#include "gz/sim/System.hh"
+
+#include "plugins/MockSystem.hh"
+#include "../test/helpers/Relay.hh"
+#include "../test/helpers/EnvTestFixture.hh"
+
+using namespace gz;
+using namespace gz::sim;
+
+/////////////////////////////////////////////////
+class ServerFixture : public InternalFixture<::testing::TestWithParam<int>>
+{
+};
+
+/////////////////////////////////////////////////
+TEST_P(ServerFixture, LoadSdfModelRelativeUri)
```

Comment:
Looks like we are not running tests that require rendering on macOS. Can you disable it for macOS for now?
```suggestion
TEST_P(ServerFixture, GZ_UTILS_TEST_DISABLED_ON_MAC(LoadSdfModelRelativeUri))
```

See 
https://github.com/gazebosim/gz-sim/blob/acd016a3ae18997443c8695213f918d44c68e638/test/integration/rgbd_camera.cc#L64